MovieApp/MovieApp/App/Coordinator/AppRouter.swift

```diff
@@ -45,7 +45,12 @@ extension AppRouter {
         let tabBarController = UITabBarController()
         
         let homePagePresenter = HomePageTwoPresenter(useCase: moviesUseCase, router: self)
-        let homePageVC = HomePageTwoViewController(presenter: homePagePresenter)
+        let categoriesPresenter = CategoriesPresenter(useCase: moviesUseCase, router: self)
+        let searchPresenter = SearchPresenter(useCase: moviesUseCase, router: self)
+        let homePageVC = HomePageTwoViewController(
```

Comment:
Could it be named HomePageViewController?

Reply:
Renamed everywhere.
[Here](https://github.com/lihter/Movie-App/pull/41/commits/3383b62f0e45eb7e1f5ba5b650a01beaa73387a1#diff-b8e8fb675a7084dca12ae6abb54b13ee1d7293192f1e1e75308b4dca15ea5465R47)